noetic/distribution.yaml

```diff
@@ -6602,6 +6602,27 @@ repositories:
       url: https://github.com/ROBOTIS-GIT/robotis_manipulator.git
       version: noetic-devel
     status: maintained
+  rom_msgs:
+    doc:
+      depends:
+      - actionlib
+      - robot_state_publisher
+      - std_msgs
+      type: git
+      url: https://github.com/greenghostman/rom_msgs.git
+      version: master
+    release:
+      packages:
+      - rom_motor_msgs
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/greenghostman/rom_msgs.git
+      version: 0.1.1-1
```

Comment:
```suggestion
```

Please remove the `release` block. It's only used when doing a binary release.

Reply:
done